### DISCUSSION TITLE
Should the Bedtime mode be removed?

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#48] [Polls]
I've been asking myself this question for several weeks.

When I talk about it to those around me, they are all unanimous: _"bedtime mode is useless"_ and I agree with them.

Bedtime mode is far from perfect in this app; some improvements need to be made and I personally don't want to spend time on it.

Before making the decision to remove this mode, I would like to get the opinions of users of this application.

**Comment by qw123wh** [comment]
I think it's better to keep it and that someone in the future could improve it. If it were possible to complete the issue #28 , the user could hide this mode or even the others that he deems unnecessary

**Reply by BlackyHawky** [reply]
Or we can create a branch dedicated to this mode. I'm not a fan of having a useless option that also needs to be greatly improved.

About #28, I've tried several times but I just can't do it. A contributor is unfortunately needed. 😢

**Comment by BlackyHawky** [comment]
The number of votes being relatively low, I made the decision to remove the "Bedtime" mode and create a branch dedicated to this mode.
This will give people the opportunity to work on this mode if they wish.
This will be done very soon before the next version.
